spec_linux.go

```diff
@@ -2,6 +2,9 @@
 
 package specs
 
+// LinuxStateDirectory holds the container's state information
+const LinuxStateDirectory = "/run/oci"
```

Comment:
i'm cool with this, as long as it's expected that it is a tmpfs state directory.